UnityProject/Assets/Scripts/Tilemaps/Behaviours/Objects/RegisterObject.cs

```diff
@@ -144,7 +146,7 @@ private bool CheckPassableExclusions(GameObject context)
 
 	#region UI Mouse Actions
 
-	public void OnHoverStart()
+	private void OnHoverStart()
```

Comment:
what is the need for these private methods? Why not just add the body of these into the implementation of the interface?

Reply:
Good point, they're currently unneeded so they can be moved directly into that function